src/components/dashboard.tsx

```
@@ -22,82 +15,17 @@ import { Button } from '@/components/ui/button';
-  'Arts & Culture': Palette,
-  'Wellness': Heart,
-};
-
-const statusIcons: Partial<Record<string, React.ComponentType>> = {
-  active: Clock,
-  completed: CheckCircle2,
-  expired: AlertCircle,
-};
-
-export default function DashboardPage({ sessionId, userId }: { sessionId: string; userId: string }) {
+export default function DashboardPage({ sessionId, user, allProjects, allDonations }: { sessionId: string; user: TUser | TOrganization; allProjects: TProject[]; allDonations: any[]; }) {
```

ESLint

[error] More than 1 blank line not allowed.

(style/no-multiple-empty-lines)